openmrs/apps/clinical/app.json

```diff
@@ -43,6 +43,9 @@
           "programAttributes": [
           ],
           "personAttributes": [
+            "givenNameLocal",
```

Comment:
`"givenNameLocal",`:
The given name is not displayed for me.
Only middle name and family name.
Does this work for you?

Also, I think it won't be right to have it displayed like that.
Ideally they should be shown next to the patient's name or something.
Having it like 'Given name ដថងុហ្ | Family name: ដថងហ' is not so nice. What do you think?

Could this be move to the 'patientInformation' dashboard widget instead of the patient header?